src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -111,11 +135,14 @@ def _call(*args):
         return compile_fx(subgraph, example_inputs)
 
 
-def fx_openvino(subgraph, example_inputs):
+def fx_openvino(subgraph, example_inputs, options):
```

Comment:
Environment variables are used in other files as well like compile.py. Could you please pass options to those files as well and have a similar solution ?

Reply:
I've searched for all path, and looks like implemented in all of them

Reply:
Hey @redradist, I was talking about this file https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py#L55 which also uses environment variables. So we need to pass the options to these functions as well to use it.

Reply:
Can you please rebase ?